### PR TITLE
comments, backward fixes

### PR DESCRIPTION
**Pull request by liorblob**
## Status
Ready

## Description
Fix context values in the old commands to their previous values

## Must have
- [x] Tests
- [ ] Code Review

### CODE REVIEW

File: Integrations/integration-ServiceNow.yml [hunk 1]
```diff
@@ -1303,7 +1331,7 @@ script:
     arguments:
     - name: id
       default: true
-      description: Ticket System ID to retrieve
```
Comment (Itay4) [inline Integrations/integration-ServiceNow.yml]: Why did you remove `to retrieve`?

File: Integrations/integration-ServiceNow.yml [hunk 2]
```diff
@@ -2926,7 +2959,8 @@ script:
     - contextPath: Ticket.Number
       description: ServiceNow ticket number
       type: string
-    description: Retrieve ticket info with query
+    description: Deprecated. Please use servicenow-query-table or servicenow-query-tickets
```
Comment (Itay4) [inline Integrations/integration-ServiceNow.yml]: Please remove `please` :)

File: Integrations/integration-ServiceNow.yml [hunk 3]
```diff
@@ -493,7 +517,8 @@ script:
                 'number': number
             }
         else:
-            raise ValueError('servicenow-get requires either ticket ID (sys_id) or ticket number')
+            # Only in cases where the table is of type ticket
+            raise ValueError('servicenow-get-ticket requires either ticket ID (sys_id) or ticket number')
```
[the remark this answers is not in the document]
Comment (liorblob) [reply Integrations/integration-ServiceNow.yml]: it catches the exceptions at the end and uses return_error, should I replace everything with `return_error` instead?

File: Integrations/integration-ServiceNow.yml (hunk 1 above)
Comment (liorblob) [reply Integrations/integration-ServiceNow.yml]: to be consistent with the other commands

File: Integrations/integration-ServiceNow.yml (hunk 2 above)
Comment (liorblob) [reply Integrations/integration-ServiceNow.yml]: @Itay4 I added that description to the old commands, this one specifically I forgot :)

File: Integrations/integration-ServiceNow.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (Itay4) [reply Integrations/integration-ServiceNow.yml]: nope
